src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -147,8 +151,8 @@ public void abortJob(JobContext context, JobStatus.State state) throws IOExcepti
     private void importFromGcs(String projectId, TableReference tableRef, @Nullable TableSchema schema,
                                @Nullable String kmsKeyName, BigQueryFileFormat sourceFormat, String writeDisposition,
                                List<String> gcsPaths, boolean allowSchemaRelaxation, boolean createPartitionedTable,
-                               @Nullable String partitionByField, boolean requirePartitionFilter)
-      throws IOException, InterruptedException {
+                               @Nullable String partitionByField, boolean requirePartitionFilter,
+                               String clusteringOrder, boolean tabelExists) throws IOException, InterruptedException {
```

Comment:
if something needs to be parsed, it's better to do it right away when it is read. So this should take a List instead of a String, with the other method reading the value from the Configuration and splitting it.

Reply:
Done.